src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -36,11 +34,10 @@ class IslandWindow : public BaseWindow<IslandWindow>
 
     winrt::Windows::UI::Xaml::Hosting::DesktopWindowXamlSource _source;
 

```

Comment:
In C++/WinRT, this will actually instantiate the DWXS object, which you probably don't mean to do at this point - so probably adding "{nullptr}" after this is what you'd want.

Same for the Grid - that's really creating a Grid object when this IslandWindow type is instantiated, which is probably earlier than you meant to.

Reply:
Oh, I see, these are set to {nullptr} in the cpp file - seems weird to not just initialize them here.

---
In reply to: [284930515](https://github.com/microsoft/terminal/pull/829#discussion_r284930515) [](ancestors = 284930515)